server/src/main/java/org/apache/uniffle/server/storage/LocalStorageManager.java

```diff
@@ -170,6 +170,11 @@ public Checker getStorageChecker() {
     return checker;
   }
 
+  @Override
+  public String getStorageUser(String appId) {
+    return null;
```

Comment:
Should we use the user who start the server?

Reply:
No need. If the user is null, fs will use the process startup user by default

Reply:
I just want to reduce `return null`. Because if we use `return null`, we should check every place to avoid NullException.

Reply:
The user only will be used in kerberos HDFS. And when in kerberos HDFS and user is null or empty, it will directly check and throw exception while flushing shuffle data.

So i think we need not to consider the NPE. Or return the empty of string?

Reply:
Empty string may be better.

Reply:
Done.